### DISCUSSION TITLE
Color balance RGB chroma slider value range

### DISCUSSION THREAD

**Discussion by pedrorrodriguez** [aurelienpierreeng/ansel#46] [Polls]
This questionnaire is an attempt at establishing a baseline among darktable/Ansel users in regards to the value ranges of the chroma sliders in the 4-ways tab. Once we have some data regarding the most common usecases, it can be used to verify whether the actual ranges are valid and useful, or if a change in a particular direction might be in order. Please notice that we are looking for the most common scenarios. 

Since github only allows for one poll per issue, here is a link to the full poll: 
https://take.supersurvey.com/QJRZDKA29

**Comment by jenshannoschwalm** [comment]
I would say we want sane values algorithm-wise. So not a question of use cases.

**Reply by pedrorrodriguez** [reply]
What do you mean? If the range of any slider is say 0-100%, but people use it mostly between 0-12%, that means that 88% of the slider range is generally useless, and what's worse, most of the time the tool is annoying to use, because you need to be extra precise, since small slider movements result in big changes. I'd say that if that's the case, it is very much about the most common use, and corner cases could be typed in manually. 

If, on the contrary, people generally use the full range of a slider that means that the value range is well set, and no change is needed. 

Either way, the survey should help clarify where darktable users fall.
